src/extensions/default/CodeFolding/unittest-files/test.js

```diff
@@ -1,33 +1,36 @@
 /**
-     * Synchronises the code folding states in the CM doc to cm._lineFolds cache.
-     * When an undo operation is done, if folded code fragments are restored, then
-     * we need to update cm._lineFolds with the fragments
-     * @param {Object}   cm       cm the CodeMirror instance for the active  editor
-     * @param {Object}   from     starting position in the doc to sync the fold states from
-     * @param {[[Type]]} lineAdded a number to show how many lines where added to the document
-     */
-    function syncDocToFoldsCache(cm, from, lineAdded) {
-        var minFoldSize = prefs.getSetting("minFoldSize") || 2;
-        var opts = cm.state.foldGutter.options || {};
-        var rf = opts.rangeFinder || CodeMirror.fold.auto;
-        var i, pos, folds, fold, range;
-        if (lineAdded <= 0) {
-            return;
-        }
+  * Synchronises the code folding states in the CM doc to cm._lineFolds cache.
+  * When an undo operation is done, if folded code fragments are restored, then
+  * we need to update cm._lineFolds with the fragments
+  * @param {Object}   cm        cm the CodeMirror instance for the active  editor
+  * @param {Object}   from      starting position in the doc to sync the fold states from
+  * @param {[[Type]]} lineAdded a number to show how many lines where added to the document
+*/
+/*global define, brackets, document, window, $, CodeMirror, isFold, prefs*/
```

Comment:
Similar to the html, this is a test file loaded up during the unit tests. the `isFold` and `prefs` are just there to silence jshint errors.